test-bundle/charmcraft.yaml

```diff
@@ -1 +1 @@
-type: bundle
+type: bundle
```

Comment:
Can this file get reverted? Looks like the newline got stripped

Reply:
Fixed